Baseledger excel

skosito opened the pull request:
# Description

Baseledger Excel uses Baseledger Lakewood (https://lakewood.baseledger.net/) testnet and Baseledger's concept TrustMesh (https://docs.baseledger.net/baseledger-concepts/trustmesh) to sync 2 Excel files. This PR introduces excel file with VBA macros code and README file with quite detailed explanation of functionality and tutorial on how to use it.

## Related Issue

Not excel specific issue, but it is mentioned here https://github.com/eea-oasis/baseline-blips/issues/6.

## Motivation and Context

Please check https://github.com/eea-oasis/baseline-blips/issues/6.

## How Has This Been Tested

It demonstrates a happy path which was tested a lot manually during development.

## Screenshots (if appropriate)

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

Only core here is VBA, which doesn't have code style defined in this project. Also, there are not automated tests for VBA code.

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

fosgate29 reviewed (changes requested):
LInk inside .xlsm https://baseledger.net/lakewoodexcel
is returning 404

skosito commented:
> LInk inside .xlsm https://baseledger.net/lakewoodexcel is returning 404

Thanks @fosgate29 great catch
I changed url now to point to README i added here - keep in mind that currently link is not working, it will after merge to master, since link contains branch information.
